tests/build-pages.test.js

```diff
@@ -0,0 +1,48 @@
+const fs = require('fs');
+const path = require('path');
+const { capitalizeJsxTags, copyAndRenameFiles } = require('../scripts/build-pages');
+
+describe('capitalizeJsxTags', () => {
+  test('should capitalize JSX tags', () => {
+    const input = '<table><tr><td>Hello</td></tr></table>';
+    const output = '<Table><Tr><Td>Hello</Td></Tr></Table>';
+    expect(capitalizeJsxTags(input)).toBe(output);
+  });
+
+  test('should not capitalize non-JSX tags', () => {
+    const input = '<div>Hello</div>';
+    const output = '<div>Hello</div>';
+    expect(capitalizeJsxTags(input)).toBe(output);
+  });
+});
+
+describe('copyAndRenameFiles', () => {
+  const TEST_DIR = 'test';
+  const SRC_DIR = path.join(TEST_DIR, 'src');
+  const TARGET_DIR = path.join(TEST_DIR, 'target');
+
+  beforeAll(() => {
+    fs.mkdirSync(TEST_DIR, { recursive: true });
+    fs.mkdirSync(SRC_DIR, { recursive: true });
+    fs.mkdirSync(TARGET_DIR, { recursive: true });
```

Comment:
`fs` module works in CI and locally in tests, then why have you mocked all those function calls in previous test PRs?